mods/default/nodes.lua

```diff
@@ -415,6 +418,21 @@ minetest.register_node("default:clay", {
 	sounds = default.node_sound_dirt_defaults(),
 })
 
+minetest.register_node("default:coral_live", {
+	description = "Coral",
+	tiles = {"default_coral_live.png"},
+	groups = {cracky = 3},
+	drop = 'default:coral',
```

Comment:
I agree, for consistency. Both work but i've gradually been changing everything to " in game.